server.py

```diff
@@ -73,6 +74,9 @@ def exposed_bv(self):
 
     def exposed_eval(self, cmd):
         return eval(cmd)
+    
+    def exposed_import_module(self, mod):
+        return importlib.import_module(mod)
```

Comment:
Actually `reload` could be useful too (maybe even more)

```suggestion
        return importlib.import_module(mod)

    def exposed_reload_module(self, mod):
        return importlib.reload(mod)
```